services/admin/src/store.ts

```diff
@@ -206,6 +221,45 @@ export class Store {
     }
   }
 
+  private convertCvrParseErrorsToApiError(
+    cvrParseResult: ParseCastVoteRecordResult
+  ): AddCastVoteRecordError {
+    // Find errors for which we want surface specific messaging to
+    // the user:
+    // [TODO: there's room for simplification here - currently leveraging
+    // older validation code in `parseCvrs` as a first step.]
+    for (const errorMessage of cvrParseResult.errors) {
+      if (errorMessage.includes('not in the election definition')) {
```

Comment:
As your comment indicates, this isn't great. It's quite fragile, but I'm okay with it for now because there are tests. We should fix this soon, though.